acds/benchmarks/mnist.py

```diff
@@ -17,13 +17,26 @@ def get_mnist_data(
         bs_test (int): Batch size for the validation and test dataloaders.
         valid_perc (int): Percentage of the train dataset to use for
             validation. Defaults to 10.
+        permuted (bool): Whether to apply permutation to images.
     """
+    permutation = torch.randperm(28 * 28) if permuted else None 
+    def permute_image(image): 
+        if permutation is not None:
+            image = image.view(-1)[permutation].view(1, 28, 28) 
```

Comment:
This looks ok, manually check that the output is correct to be extra sure

Reply:
Checked